ast/builtins.go

```diff
@@ -1418,6 +1419,19 @@ var UUIDRFC4122 = &Builtin{
 	Nondeterministic: true,
 }
 
+var UUIDParse = &Builtin{
+	Name:        "uuid.parse",
+	Description: "Parses the string value as an UUID and returns an object with the well-defined fields of the UUID if valid.",
+	Categories:  nil,
+	Decl: types.NewFunction(
+		types.Args(
+			types.Named("uuid", types.S),
+		),
+		types.Named("result", types.NewObject(nil, types.NewDynamicProperty(types.S, types.A))).Description("properties of UUID if valid (version, variant, etc). Undefined otherwise "),
```

Comment:
[nit] [This is what gets rendered based on these annotations](https://deploy-preview-6208--openpolicyagent.netlify.app/docs/edge/policy-reference/#builtin-uuid-uuidparse). So, maybe let's use this?
```suggestion
		types.Named("result", types.NewObject(nil, types.NewDynamicProperty(types.S, types.A))).Description("Properties of UUID if valid (version, variant, etc). Undefined otherwise."),
```

Reply:
Oooh, that is better. Fixed the typos and `make generate`-ed it to reflect this